### PR TITLE
BL-8921 security checks

### PR DESCRIPTION
**Pull request by adrianz-kainos**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -24,14 +25,14 @@
...
-    "cvr-common": "file:../common",
     "aws-sdk": "^2.338.0",
     "aws-serverless-express": "^3.3.5",
     "body-parser": "^1.18.3",
+    "cvr-common": "file:../common",
     "express": "^4.16.4",
     "express-http-context": "^1.2.0",
-    "promise-sequential": "^1.1.1",
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
```
Comment (zakrzewskam) [inline backend/package.json]: Cleanup required.

File: common/package.json [hunk 2]
```diff
@@ -15,11 +16,12 @@
     "safe-json-stringify": "^1.2.0"
   },
   "devDependencies": {
-    "mocha": "^5.2.0",
     "chai": "^4.2.0",
-    "sinon": "^7.1.1",
     "eslint": "^5.7.0",
     "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.14.0"
+    "eslint-plugin-import": "^2.14.0",
```
Comment (zakrzewskam) [inline common/package.json]: Cleanup required.

File: data-update/package.json [hunk 3]
```diff
@@ -25,11 +26,12 @@
     "request": "^2.88.0"
   },
   "devDependencies": {
-    "mocha": "^5.2.0",
```
Comment (zakrzewskam) [inline data-update/package.json]: Cleanup required.

File: database/package.json [hunk 4]
```diff
@@ -17,7 +18,8 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "mocha": "^5.2.0",
```
Comment (zakrzewskam) [inline database/package.json]: Cleanup required.

File: frontend/package.json [hunk 5]
```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "start": "nodemon ./scripts/local.js --watch ../common --watch ./",
     "debug": "nodemon --nolazy --inspect-brk=8228 ./scripts/local.js --ext js,njk --watch ../common --watch ./",
-    "test": "export APP_NAME=test && nyc mocha \"test/**/*.test.js\" --exit",
+    "test": "retire && export APP_NAME=test && nyc mocha \"test/**/*.test.js\" --exit",
```
Comment (zakrzewskam) [inline frontend/package.json]: Should we run retire as a part of tests? I think it should be separated job, especially when vulnerable dependencies you checking only once, but tests you sometimes runs few times.

The same comment for each of the sub-packages.

**Comment by zakrzewskam** [comment]
You should add the information about the vulnerable dependencies check to README.md

File: frontend/package.json (hunk 5 above)
Comment (adrianz-kainos) [reply frontend/package.json]: Running it in the "build" script should be enough. I will remove it from tests.

File: README.md [hunk 6]
```diff
@@ -76,4 +79,4 @@ npm run build:<module_name>
 
 * Assets are deployed based on [assets.verison file](./frontend/assets.version). More on recalls can be found in *Recalls home* internal wiki. 
 * You can push a feature branch without passing lint/tests by adding `--no-verify` to your `git push` command
-* You can run `npm run clean` if you need all node_module folders deleted should you need it
\ No newline at end of file
+* You can run `npm run clean` if you need all node_module folders deleted should you need it
```
Comment (zakrzewskam) [inline README.md]: ?

Comment (adrianz-kainos) [reply README.md]: My IDE automatically adds a newline to a file if it does not have one at the end. That's why you are seeing it

File: README.md [hunk 7]
```diff
@@ -64,6 +64,9 @@ You can update the package lockfile in all modules via `npm run update-package-l
 ## Running full validation
 To simplify things you can run all checks on all modules by running `npm run validate`. This should ensure all module tests and code quality checks pass.
 
+## Checking dependencies for vulnerabilities
```
Comment (zakrzewskam) [inline README.md]: I think this point should be after lint checks. Moreover "Running full validation" require updating.

**Pull request merged**